media-sound/mixxx/mixxx-9999.ebuild

```
@@ -14,119 +14,156 @@ if [[ ${PV} == *9999 ]] ; then
 	fi
 	EGIT_REPO_URI="https://github.com/mixxxdj/${PN}.git"
 else
-	SRC_URI="https://github.com/mixxxdj/${PN}/archive/release-${PV}.tar.gz -> ${P}.tar.gz"
-	S="${WORKDIR}/${PN}-release-${PV}"
-	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/mixxxdj/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
 fi
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="aac doc ffmpeg hid keyfinder lv2 mp3 mp4 opus qtkeychain shout wavpack"
+IUSE="aac benchmark ffmpeg keyfinder lv2 midi modplug mp3 mp4 opus qml"
+IUSE+=" qtkeychain rubberband shout test upower wavpack"
+REQUIRED_USE="
+	benchmark? ( test )
+	qtkeychain? ( shout )
+	test? ( aac ffmpeg midi mp3 opus rubberband )
+"
+RESTRICT="!test? ( test )"
 
 RDEPEND="
-	dev-db/sqlite
-	dev-libs/glib:2
+	dev-db/sqlite:3
+	dev-libs/hidapi
 	dev-libs/protobuf:=
-	dev-qt/qtconcurrent:5
-	dev-qt/qtcore:5
-	dev-qt/qtdbus:5
-	dev-qt/qtgui:5
-	dev-qt/qtnetwork:5
-	dev-qt/qtopengl:5
-	dev-qt/qtscript:5[scripttools]
-	dev-qt/qtsql:5
-	dev-qt/qtsvg:5
-	dev-qt/qtwidgets:5
-	dev-qt/qtx11extras:5
-	dev-qt/qtxml:5
-	media-libs/chromaprint
+	dev-qt/qt5compat:6
```

```
$ grep -r GraphicalEff 
CMakeLists.txt:        DIRECTORY "${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}$<$<CONFIG:Debug>:/debug>/Qt6/qml/Qt5Compat/GraphicalEffects"
res/qml/Button.qml:import Qt5Compat.GraphicalEffects
res/qml/DeckInfoBar.qml:import Qt5Compat.GraphicalEffects
res/qml/InfoBarButton.qml:import Qt5Compat.GraphicalEffects
res/qml/Slider.qml:import Qt5Compat.GraphicalEffects
res/qml/VuMeter.qml:import Qt5Compat.GraphicalEffects
```

...means we need at least in RDEPEND:
```suggestion
	dev-qt/qt5compat:6[qml]
```